releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,9 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 6.0.1 - May 3, 2023
+* Fixed a bug where .NET Framework applications would fail while trying to refresh the configuration. [#410](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/410)
```

Comment:
... where ASP.NET (.NET Framework) applications...

I don't think this can be repro'ed in a console app.


... fail ~while trying~ to refresh...

The application didn't fail. Just the refresh failed.